cmd/sealos/cmd/pull.go

```diff
@@ -37,6 +38,9 @@ func newPullCmd() *cobra.Command {
 }
 
 func init() {
+	if buildah.InitReexec() {
```

Comment:
buildah init  can move to servic interface new func

Reply:
main func already has call this func?

Reply:
I fix it.this func already has call.